docs/docs/installation_guide/installation/pre-installation/ec2-instance-req.md

```diff
@@ -1,7 +1,7 @@
 ---
-id: instance-req
-title: Instance requirements
-sidebar_label: Instance requirements
+id: ec2-instance-req
+title: EC2 Instance requirements
```

Comment:
@dengmingtong do you think it's necessary?

Reply:
I think it's better to keep title and sidebar.

Reply:
The modification here is mainly to make the title in the table consistent with the title in the sidebar.

<img width="1176" alt="image" src="https://user-images.githubusercontent.com/106293323/175471543-8e72289a-46ff-42df-962b-9bd2fa7ccc7e.png">

Reply:
It's something I prefer to keeping sync with upstream.

Could you submit a PR to fix the table style in upstream. Also create a bug for section title and sidebar mismatch. Let's see the feedback from upstream.

Reply:
I created this [PR](https://github.com/awslabs/service-workbench-on-aws/pull/1006) and will update here as soon as there are any comments.